packages/shared/src/types/styleguide.types.ts

```
@@ -0,0 +1,37 @@
+  StyleguideSectionId.Buttons,
+];
+
+export class Styleguide {
+  public static getOrderedSectionIds(): StyleguideSectionId[] {
+    return ORDERED_STYLEGUIDE_SECTION_IDS;
+  }
+
+  public static getSectionById(styleguideSectionId: StyleguideSectionId): StyleguideSectionConfig {
+    return STYLEGUIDE_SECTIONS_BY_ID[styleguideSectionId];
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)